package.json

```diff
@@ -1,28 +1,40 @@
 {
-  "name": "electron-sentry",
-  "description": "JavaScript and native crash reporting from Electron to Sentry",
-  "version": "1.1.2",
```

Comment:
This will need to be greater than the last to publish to npm. 

As the entire API has changed v2 would be preferable!